src/stratis_cli/_actions/_logical.py

```diff
@@ -98,11 +98,14 @@ def list_volumes(namespace):
         List the volumes in a pool.
         """
         # pylint: disable=import-outside-toplevel
+        from ._constants import FILESYSTEM_INTERFACE
+        from ._data import FetchProperties
         from ._data import MOFilesystem
         from ._data import MOPool
         from ._data import ObjectManager
         from ._data import filesystems
         from ._data import pools
+        from ._formatting import fetch_property
```

Comment:
Here's another import that I'ld like also like  to be at the top...completely missed that the last time.